scene/register_scene_types.cpp

```diff
@@ -1268,6 +1268,17 @@ void unregister_scene_types() {
 	ResourceLoader::remove_resource_format_loader(resource_loader_shader_include);
 	resource_loader_shader_include.unref();
 
+	// Cleanup scene property helper singletons.
+	{
+		FileDialog::clear_base_property_helper();
+		ItemList::clear_base_property_helper();
+		MenuButton::clear_base_property_helper();
+		OptionButton::clear_base_property_helper();
+		PopupMenu::clear_base_property_helper();
+		TabBar::clear_base_property_helper();
+		TileMap::clear_base_property_helper();
+	}
```

Comment:
Looks ok overall, but I'd love to find a solution that doesn't require hardcoding these calls for each class using PropertyListHelper.

Would like @KoBeWi's opinion on this, and ideas for alternative approaches, if there's any.

Reply:
@KoBeWi Done, added static vector of instances which cleaned at the main